bindata/network/ovn-kubernetes/ovnkube-master.yaml

```diff
@@ -82,6 +82,8 @@ spec:
         env:
         - name: OVN_LOG_LEVEL
           value: info 
+        - name: OVN_LOG_PATTERN_CONSOLE
```

Comment:
Can you just declare this pattern as a constant in https://github.com/openshift/cluster-network-operator/blob/master/pkg/network/ovn_kubernetes.go#L46

rather than re declaring it for each container? Otherwise LGTM

Reply:
Sure. Yes this is way better! I updated it...